src/typechecker.erl

```diff
@@ -1127,7 +1127,7 @@ expect_fun_type_union(Env, [Ty|Tys]) ->
             [TyOut | expect_fun_type_union(Env, Tys)]
     end.
 
-expect_record_type(Record, {type, _, record, [{atom, _, Name}]}, _TEnv) ->
+expect_record_type(Record, {type, _, record, [{atom, _, Name}|_]}, _TEnv) ->
```

Comment:
In general, please add a TODO comment when adding code to ignore unimplemented stuff.